clang-tools-extra/clangd/XRefs.cpp

```diff
@@ -430,6 +439,22 @@ locateASTReferent(SourceLocation CurLoc, const syntax::Token *TouchedIdentifier,
         continue;
       }
     }
+    // Special case: - (void)^method; should jump to all overrides. Note that an
+    // Objective-C method can override a parent class or protocol.
```

Comment:
i think it's worthwhile to also add a test to check we don't return overridden methods when calling xrefs on usage of the method.

Reply:
Done.